dsc_lib/src/discovery/command_discovery.rs

```diff
@@ -496,3 +507,96 @@ fn load_manifest(path: &Path) -> Result<DscResource, DscError> {
 
     Ok(resource)
 }
+
+fn sort_adapters_based_on_lookup_table(unsorted_adapters: &BTreeMap<String, Vec<DscResource>>, needed_resource_types: &Vec<String>) -> LinkedHashMap<String, Vec<DscResource>>
+{
+    let mut result:LinkedHashMap<String, Vec<DscResource>> = LinkedHashMap::new();
+    let lookup_table:HashMap<String, String> = load_adapted_resources_lookup_table();
+    // first add adapters (for needed types) that can be found in the lookup table
+    for needed_resource in needed_resource_types {
+        if let Some(adapter_name) = lookup_table.get(needed_resource) {
+            if let Some(resource_vec) = unsorted_adapters.get(adapter_name) {
+                trace!("Lookup table found resource '{}' in adapter '{}'", needed_resource, adapter_name);
```

Comment:
We should use `debug!` more for traces that don't contain private data like this line